crates/uv/tests/it/build.rs

```diff
@@ -1799,6 +1800,7 @@ fn build_with_hardlink() -> Result<()> {
         .temp_dir
         .child("src/hardlinked/__init__.py")
         .touch()?;
+    fs_err::remove_dir_all(&context.venv)?;
```

Comment:
@konstin -- It looks like in this case, we're including the `.venv` in the sdist. I'm not sure why. Doesn't hatchling typically filter out `.gitignore`?